test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored
```

Comment:
- [ ] [OPTIONAL] GitHub isn't ignoring your file. The reason might be that you pushed the file first before adding it to `.git ignore file.` Try deleting it. then push it to GitHub. After that, create the file again and make sure it's in the `.gitignore` file before pushing it to GitHub.